components/scream/src/interface/input.yaml

```diff
@@ -0,0 +1,21 @@
+%YAML 1.1
```

Comment:
Is scream/src/interface/ the right place for this? In particular, a yaml file doesn't seem like a source file to me. Maybe we should ask Jim where to put yaml files?

Reply:
It is not. The `scream_mct_interface` branch introduces a default `scream_input.yaml` file in `scream/data`. I'm talking to aaron, to get our two branches integrated before they go in master. Haven't decided which direction is best, but they should definitely be combined,since they both involve mct interfaces.